dev-libs/antlr-c/antlr-c-3.5.2-r2.ebuild

```diff
@@ -42,24 +39,23 @@ multilib_src_configure() {
 		*) econfargs+=( --disable-64bit ) ;;
 	esac
 
-	econf "${econfargs[@]}"
+	CONFIG_SHELL="${BROOT}"/bin/bash econf "${econfargs[@]}"
```

Comment:
My personal preference would be to `local -x ...` it instead of long inline.

Reply:
While I might adopt that in future, I'm reluctant to make further style changes for this PR.